src/app/page.tsx

```
@@ -15,6 +18,11 @@ export default async function MainPage({ searchParams }: MainPageProps) {
   const _searchParams = new URLSearchParams(searchParams);
   const sortBy = getSortByValue(_searchParams.get('sortBy'));
   const sortOrder = getSortOrderValue(_searchParams.get('sortOrder'));
+
+  const cookieStore = cookies();
+  const token = cookieStore.get('access_token');
+  const user = !!token?.value ? await getUser() : null;
```

Biome (1.9.4)

[error] Avoid redundant double-negation.

It is not necessary to use double-negation when a value will already be coerced to a boolean.
Unsafe fix: Remove redundant double-negation


(lint/complexity/noExtraBooleanCast)